README.md

```diff
@@ -21,12 +21,18 @@ pip install -r requirements.txt
 # Configure the flask application
 export FLASK_APP=manage
 # Setup the database
-flask db init
+docker run --name dataservice-pg -p 5432:5432 -d postgres:9.5
+docker exec dataservice-pg psql -U postgres -c "CREATE DATABASE dev;"
 flask db upgrade
```

Comment:
Need to add a line above here with `flask db migrate`, otherwise the relations don't exist on the first time you try to run the app